### PR TITLE
#167729775-Ch-Integrating-Travis-CI

### PR DESCRIPTION
**Pull request by davidfrank96**
#### What does this PR do?
Integrated Travis CI for continuous integration for deploying with confidence 

#### Description of Task to be completed?
- implemented a travis.yml file for Travis cloud to detect

#### How should this be manually tested?
pull or fetch this branch  
run npm install to add necessary dependencies

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
ch-Integrating-Travis-ci-167729775
#### Screenshots (if appropriate)
Travis

### CODE REVIEW

**Review by okabamac — APPROVED** [review]
Looks good to me and Travis is now working. Good job

**Review by chibytez — APPROVED** [review]
gone through it, looks great, please notify others of the port and secret key

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+before_install:
+ - npm install sequelize-cli -g
+
+env:
+ global:
+   - TEST_DATABASE_URL=postgres://postgres:@127.0.0.1:5432/test_database
+   - SECRET_KEY=secretkey
+   - NODE_ENV=test
+   - SOCKET_PORT=3000
```
Comment (Stylll) [inline .travis.yml]: @davidfrank96 We should not be displaying our global variables like this. Travis provides a key/value pair store where these variables can be stored and then referenced here. Please look into that.

**Review by Stylll — CHANGES_REQUESTED** [review]
Hey @davidfrank96 Please check out my comment.

File: .travis.yml (hunk 1 above)
Comment (davidfrank96) [reply .travis.yml]: thank you for the kind words.. will implement that ASAP

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ - postgresql
+
+before_install:
+ - npm install sequelize-cli -g
+   
+
+before-script:
+ - npm add sequelize-cli
+
+before_script:
```
Comment (Stylll) [inline .travis.yml:14]: @davidfrank96 You have two before_script sections. Can you combine them into 1?

**Review by Stylll — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (davidfrank96) [reply .travis.yml:14]: Okay...

**Review by Stylll — APPROVED** [review]

**Pull request merged**